config/initializers/ruby_stackoverflow.rb

```diff
@@ -0,0 +1,4 @@
+RubyStackoverflow.configure do|config|
+  config.client_key = ENV["CLIENT_KEY"]
```

Comment:
Prefix these environment variables so that they look like `"STACKEXCHANGE_CLIENT_KEY"`